internal/blockdb/sql.go

```diff
@@ -4,23 +4,33 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
+	"os"
+	"path/filepath"
 
 	_ "modernc.org/sqlite"
 )
 
-// ConnectDB connects to the sqlite database at filePath with max connections set to maxConns.
+// ConnectDB connects to the sqlite database at databaseFile.
 // Pings database once to ensure connection.
-// Pass :memory: as filePath for in-memory database.
-func ConnectDB(ctx context.Context, filePath string, maxConns int) (*sql.DB, error) {
-	db, err := sql.Open("sqlite", filePath)
+// Creates directory path via MkdirAll.
+// Pass :memory: as databaseFile for in-memory database.
+func ConnectDB(ctx context.Context, databaseFile string) (*sql.DB, error) {
```

Comment:
Can the parameter `databaseFile` be renamed to `databasePath` since the string is a path, not a file object?